### ISSUE TITLE
No mention in release notes

### ISSUE THREAD

**Issue by pandrien** [mozilla-services/topsites-proxy#58]
https://www.mozilla.org/en-US/firefox/83.0/releasenotes/

For a privacy impacting opt-out feature like this, I think there is generally an expectation that it should be included in the release notes. The omission creates the perception that you are trying to hide this update, and has understandably upset users.

**Comment by Nekkowe** [comment]
This is absolutely appalling.

**Comment by garymoon** [comment]
Mozilla: What are you **_doing_** :sob: :man_facepalming:

**Comment by huynhxloi** [comment]
Agreed - just saw Ebay randomly popping up in my Top Sites section, pushing one of my other Top Sites out of the way. I read every Release Notes after every update and never recalled seeing anything there about this "feature". It is NOT a "feature" and the fact that it was snuck in and enabled itself without my knowledge/permission is not acceptable.

**Comment by pachainti** [comment]
Any update?
